cluster-autoscaler/FAQ.md

```diff
@@ -177,3 +202,9 @@ There are three options:
     * on pods (particularly those that cannot be scheduled).
     * on nodes.
     * on kube-system/cluster-autoscaler-status config map.
+
+### What happens in scale up when I have no more quota in the cloud provider?
+
+Scale up will periodically try to increase the cluster and move back to the previous size until the quota arrives or
+the scale-up-triggering pods are removed.
```

Comment:
This reads wrong to me. Scale up will try to A (increase cluster) and B (move back to prev size) reads like it's trying to do 2 things, as opposed to B being the result of A failing.